po/pl.po

```diff
@@ -18,1295 +20,1298 @@ msgstr ""
 
-#: Settings.ui.h:210
-msgid ""
-"LeftBox Font Size\n"
-"(0 = theme default)"
-msgstr ""
-"Rozmiar czcionki lewej strony panelu\n"
-"(0 - domyślne motywu)"
+#: Settings.ui.h:228
+msgid "LeftBox Font Size\n(0 = theme default)"
+msgstr "Rozmiar czcionki lewej strony panelu\n(0 - domyślne motywu)"
 
-#: Settings.ui.h:212
-msgid ""
-"Tray Item Padding\n"
-"(-1 = theme default)"
-msgstr ""
-"Odstęp elementów zasobnika\n"
-"(-1 - domyślne motywu)"
+#: Settings.ui.h:229
+msgid "Tray Item Padding\n(-1 = theme default)"
+msgstr "Odstęp elementów zasobnika\n(-1 - domyślne motywu)"
 
-#: Settings.ui.h:214
-msgid ""
-"Status Icon Padding\n"
-"(-1 = theme default)"
-msgstr ""
-"Odstęp elementów menu systemowego\n"
-"(-1 - domyślne motywu)"
+#: Settings.ui.h:230
+msgid "Status Icon Padding\n(-1 = theme default)"
+msgstr "Odstęp elementów menu systemowego\n(-1 -domyślne motywu)"
 
-#: Settings.ui.h:216
-msgid ""
-"LeftBox Padding\n"
-"(-1 = theme default)"
-msgstr ""
-"Odstęp elementów lewej strony panelu\n"
-"(-1 - domyślne motywu)"
+#: Settings.ui.h:231
+msgid "LeftBox Padding\n(-1 = theme default)"
+msgstr "Odstęp elementów lewej strony panelu\n(-1 - domyślne motywu)"
 
-#: Settings.ui.h:218
+#: Settings.ui.h:232
 msgid "Animate switching applications"
-msgstr "Animuj przełączenie programów"
+msgstr "Animuj przełączanie programów"
 
-#: Settings.ui.h:219
+#: Settings.ui.h:233
 msgid "Animate launching new windows"
 msgstr "Animuj uruchamianie nowych programów"
 
-#: Settings.ui.h:220
+#: Settings.ui.h:234
 msgid "Keep original gnome-shell dash (overview)"
-msgstr "Zachowaj oryginalny panel (podgląd)"
+msgstr "Zachowaj orginalny panel (podgląd)"
 
-#: Settings.ui.h:221
+#: Settings.ui.h:235
+msgid "Force Activities hot corner on primary monitor"
+msgstr "Wymuś działanie Aktywnego rogu na głównym monitorze"
+
+#: Settings.ui.h:236
 msgid "Activate panel menu buttons (e.g. date menu) on click only"
 msgstr ""
 "Aktywuj przyciski menu panelu (np. menu kalendarza) tylko po kliknięciu"
 
-#: Settings.ui.h:222
+#: Settings.ui.h:237
+msgid "Keep original gnome-shell top panel"
+msgstr "Zachowaj orginalny górny panel"
+
+#: Settings.ui.h:238
 msgid "App icon secondary (right-click) menu"
 msgstr "Menu kontekstowe programu (prawy przycisk myszy)"
 
-#: Settings.ui.h:224
+#: Settings.ui.h:240
 msgid "Fine-Tune"
 msgstr "Dostrajanie"
 
-#: Settings.ui.h:225
+#: Settings.ui.h:241
 msgid "version: "
-msgstr "wersja: "
+msgstr ""
 
-#: Settings.ui.h:226
+#: Settings.ui.h:242
 msgid "GitHub"
-msgstr "GitHub"
+msgstr ""
 
-#: Settings.ui.h:227
+#: Settings.ui.h:243
 msgid ""
 "Use the buttons below to create a settings file from your current "
 "preferences that can be imported on a different machine."
 msgstr ""
-"Użyj poniższych przycisków, aby utworzyć plik ustawień bieżącychpreferencji, "
-"które można zaimportować na innym komputerze."
+"Użyj poniższych przycisków, aby otworzyć plik ustawień bieżących "
+"preferencji, które można zaimportować na innym komputerze"
```

Comment:
"create" translates to "utworzyć", so it was correct before. I'd change "które" to "który", because it refers to the file, not to the settings.